TakeOne/Assets/_Project/Scripts/Relics/RelicSO.cs

```diff
@@ -0,0 +1,15 @@
+using UnityEngine;
+
+namespace DiceGame.Relics
+{
+    [CreateAssetMenu(menuName = "Create RelicSO", fileName = "RelicSO", order = 0)]
+    public class RelicSO : ScriptableObject
```

Comment:
There's more going into the RelicSO right?

Reply:
yes